irispy/tests/test_new_sji.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# """Tests for functions in sji.py"""
+import datetime
+
+import pytest
+import numpy as np
+from ndcube.utils.wcs import WCS
+
+from irispy import iris_tools
+from irispy.new_sji import SJICube, read_iris_sji_level2_fits
+
+# Sample data for tests
+private_fits_path = '/mn/stornext/u3/baptistp/iris/level2/2014/12/11/20141211_191222_3803105278/'
+file_name = 'iris_l2_20141211_191222_3803105278_SJI_1330_t000.fits'
+file_path = private_fits_path + file_name
```

Comment:
This raises an important point that we should discuss, i.e. whether we should store files of test data in the repo. @Cadair has had this discussion a few times in SunPy.  Any thoughts @Cadair?

Reply:
I think in order to get this PR merged, I'd suggest removing this definition of the FITS file and also removing the ```test_read_iris_sji_level2_fits```.  Then you can submit a new PR with ```test_read_iris_sji_level2_fits``` and we can deal with the issue of test data files separately.